spaceship/lib/spaceship/connect_api/models/profile.rb

```diff
@@ -71,6 +73,14 @@ def self.all(filter: {}, includes: nil, limit: nil, sort: nil)
       end
 
       def self.create(name: nil, profile_type: nil, bundle_id_id: nil, certificate_ids: nil, device_ids: nil, template_name: nil)
+        # map direct_kext to direct
+        case profile_type
+        when ProfileType::MAC_APP_DIRECT_KEXT
+          profile_type = ProfileType::MAC_APP_DIRECT
+        when ProfileType::MAC_CATALYST_APP_DIRECT_KEXT
+          profile_type = ProfileType::MAC_CATALYST_APP_DIRECT
+        end
+
```

Comment:
One quick question! How come we need this map? 🤔 Don't we want to create `ProfileType::MAC_APP_DIRECT_KEXT` and `ProfileType::MAC_CATALYST_APP_DIRECT_KEXT`?

Reply:
Ping @langyanduan 🤗

Reply:
@langyanduan are you still looking to get this merged? 😊 😊 
If yes, it would be awesome to rebase master, fix lint and answer the question 🚀